frontend/src/views/supp-allowances/SupplementaryFormView.vue

```diff
@@ -7,21 +7,29 @@
   </p>
 
   <AppAlertBanner v-if="currentTermDisabled" type="warning">
-    <div>Your current term funding is ending and you are no longer able to make changes. Please apply for Next Term</div>
+    <div>Your current year funding is ending and you are no longer able to make changes. Please apply for Next Year</div>
   </AppAlertBanner>
 
   <v-form ref="form">
     <v-row no-gutters class="mb-2">
       <v-col cols="12" lg="1" class="">
-        <AppLabel>Application Term:</AppLabel>
+        <AppLabel>Application Year:</AppLabel>
       </v-col>
 
       <!-- These buttons should always be enabled/disabled with FA dates as we might not have a supp app date to go off of-->
       <v-col cols="6" lg="2" class="d-flex flex-column align-end">
-        <AppButton id="current-term-button" class="mr-1" :active="!nextTermActive" :primary="false" size="large" width="200px" @click="setActiveTerm(false)">Current Term</AppButton>
+        <AppButton id="current-year-button" class="mr-1" :active="!nextTermActive" :primary="false" size="large" width="200px" @click="setActiveTerm(false)">Current Year</AppButton>
       </v-col>
       <v-col cols="6" lg="2" class="">
-        <AppButton id="next-term-button" :active="nextTermActive" :primary="false" :disabled="!isNextTermEnabled" size="large" width="200px" @click="setActiveTerm(true)">Next Term</AppButton>
+        <AppButton id="next-year-button" :active="nextTermActive" :primary="false" :disabled="!isNextTermEnabled" size="large" width="200px" @click="setActiveTerm(true)">Next Year</AppButton>
+      </v-col>
+    </v-row>
+    <v-row v-if="fundingExpiryDate">
+      <v-col cols="12" class="">
+        <div v-if="!nextTermActive">
+          If you apply for and receive funding in the current year of your funding agreement, the funds must be used by
+          {{ new Date(Date.parse(fundingExpiryDate.toISOString())).toLocaleString('en-US', { timeZone: 'UTC', dateStyle: 'full' }) }}
```

Comment:
I think we can add this one to format.js, so we can reuse it in other components? also, we normally use locale 'en-CA' (although I think 'en-US' and 'en-CA')

Reply:
yes good call moving it to format.js - and I also have changed it to use CA instead :)